Backend/tests/db_utils.py

```diff
@@ -64,8 +68,10 @@ def preloadData(jsonData, table):
           }
         )
   except IOError as e:
-    print("Unable to load data into table")
-    print(e)
+    logger.exception(e)
+    message = "Unable to load data into table"
+    logger.error(message)
```

Comment:
Could we combine these since they are related? like logger.exception(message  + "\n" + e) or the python string equivalent of that